### DISCUSSION TITLE
Archive without rsync

### DISCUSSION THREAD

**Discussion by leonidcan** [ericaltendorf/plotman#604] [Q&A]
It seems on my system some plots are made to dst in last phase (what is the reason for that?) instead of using local fast drive. As a result it takes very long time to complete plots. Is it possible to do archive stage using some copy method? I am windows guy and my farmer is windows machine. I tried to use ubuntu but my external storage has idle shutdown which I can't disable and it's disabled on Windows ???

**Comment by altendky** [comment]
I'm not completely certain what all problems you are having but we are working on an archive setup that will allow customization of the commands used to transfer the files.

**Reply by lnb2g** [reply]
the problem with rsync is that its too slow. Most of people come to the conclusion that they can use 2nd and dst as the same drive which speed up a process a little. The problem is when it is the final plot will stay in that 2nd drive. It would be nice to have a copy function as an archive option. I am thinking to modify the archive.py to achieve this but im not a coder and had google alot of stuff to put them together which is suck.
